partcad.yaml

```diff
@@ -30,6 +30,22 @@ parts:
       corner-bottom-back-right: [[85, 56, 0], [0, 0, 1], 180]
       corner-bottom-front-left: [[0, 0, 0], [0, 0, 1], 0]
       corner-bottom-front-right: [[85, 0, 0], [0, 0, 1], 90]
+  rpi-touchscreen:
```

Comment:
Does it make sense to call it "shields/rpi-touchscreen" and drop the "path" attribute?

Reply:
"shields/touchscreen"

Reply:
I suggest to leave it as it is now and decide if we want to refactor this later